examples/with-eip-1193-provider/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
This README needs to be updated too! You can copy an example from another example folder: it should explain the purpose of the demo, how to get the proper env vars, and maybe link to resources if relevant